docs/proposals/kubelet-oci-runtime.md

```diff
@@ -0,0 +1,87 @@
+Kubelet OCI runtime integration
+===========================================================
+Authors: Mrunal Patel (@mrunalp), Vishnu Kannan (@vishh)
+
+## Abstract
+This proposal aims to introduce support for OCI Runtime Specification compatible runtimes in Kubelet.
+
+## Motivation
+Kubelet should support runc and OCI as a container runtime to support Open Standards. Runc is the OCI compliant
```

Comment:
nit: The OCI has not defined a compliance program so nothing can claim to be OCI compliant. And even if it has runc is only one potential OCI implementation, it is not "the Linux implementation". See: https://github.com/projectatomic/bubblewrap, github.com/wking/ccon